src/ol/format/WFS.js

```diff
@@ -143,13 +143,23 @@ const TRANSACTION_SERIALIZERS = {
  * @property {number} [count] Number of features to retrieve when paging. This is a
  * WFS 2.0 feature backported to WFS 1.1.0 by some Web Feature Services. Please note that some
  * Web Feature Services have repurposed `maxfeatures` instead.
- * @property {import("../extent.js").Extent} [bbox] Extent to use for the BBOX filter.
+ * @property {import("../extent.js").Extent} [bbox] Extent to use for the BBOX filter. The `geometryName`
+ * option must be set.
+ * @property {Array<FeatureTypeBbox>} [featureTypesBbox] Define a unique bbox filter per feature type
```

Comment:
I think `featureTypes` would be a better name for the property. And the `FeatureTypeBbox` type could be renamed to `FeatureType`.

Reply:
The problem is that there is already a property named `featureTypes` in this object.
```
* @property {Array<string>} featureTypes The feature type names.
```
What should I do ?

(By the way, `featureTypes` is now optional, I've to write it `[featureTypes]` here)

Reply:
Should not be a problem. `featureTypes` can either be an `Array<string>` or an `Array<FeatureType>`.

Reply:
Done thanks :-)